src/tests/ftest/cart/ctl/cart_ctl_five_node.yaml

```diff
@@ -23,10 +22,8 @@ hosts: !mux
       - boro-C
       - boro-D
       - boro-E
-    test_clients_1:
-      - boro-A
-    test_clients_2:
-      - boro-A
+    test_clients:
+      - boro-F
```

Comment:
Per @sylviachanoiyee - Here we've skipped A and B.  Label nodes ascending alphabetically from A-Z.